karma.conf.js

```diff
@@ -20,12 +20,17 @@ module.exports = function (karma) {
 				'Promise',
 				'matchMedia',
 				'HTMLPictureElement',
-				// the following polyfills are included pending https://github.com/Financial-Times/polyfill-service/issues/653
-				'CustomEvent|always|gated',
-				'fetch|always|gated',
-				'Array.prototype.find|always|gated',
-				'Array.prototype.findIndex|always|gated'
-			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set',
+				'fetch',
+				'Array.prototype.find',
+				'Array.prototype.findIndex',
+				'IntersectionObserver',
+				'Map',
+				'Array.from',
+				'NodeList.prototype.@@iterator',
+				'Array.prototype.@@iterator',
+				'EventSource',
+				'Number.isInteger'
+			].join(',') + '&flags=gated&source=n-syndication-tests',
```

Comment:
Was getting polyfill errors in IE. I'm not _quite_ sure why it was including a weird set of polyfills - seems to make more sense to keep it the same as what's on the site.

But as a result...